vast/integration/vast_integration_suite.yaml

```diff
@@ -592,3 +592,14 @@ tests:
         expected_result: ignore
       - command: export --timeout=800ms json
         expected_result: ignore
+
+  Self Repair:
+    steps:
+      - command: -N import zeek
+        input: data/zeek/conn.log.gz
+        transformation: "@./misc/scripts/break-sizelimit.sh"
```

Comment:
Isn't `transformation` just piping it to this command? Is it guaranteed that the files are already written to disk when the script runs? If not, we can insert an additional `vast flush` step to attach the transformation to.

Reply:
We're using `vast -N` here, so the partition is guaranteed to exist.

Reply:
I don't see why? I thought this would expand to the equivalent of

```
vast -N import zeek | ./misc/scripts/break-sizelimit.sh
```

Reply:
You are right! I totally missed that. I'll fix.

Reply:
Oh, I see you already added another step. Nevermind then.